src/tree/tree.scss

```diff
@@ -0,0 +1,149 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../icon/index';
+
+$iui-active-outline: thin solid t(iui-color-foreground-primary);
+
+// Spacing for when expander button is not present: (icon margin 6px) + (icon width 16px) + (button padding 4px) + (button border 2px)
+$iui-expander-button-width: 28px;
+
+@mixin iui-tree {
+  @include themed {
+    background-color: t(iui-color-background-1);
+  }
+  list-style: none;
+  @include iui-reset;
+}
+
+@mixin iui-sub-tree {
+  list-style: none;
+  @include iui-reset;
+}
+
+@mixin iui-tree-node {
+  display: flex;
+  cursor: pointer;
+  padding: 0 $iui-s;
+
+  &-checkbox {
+    margin-right: $iui-s;
+  }
+
+  &-content {
+    display: flex;
+    align-items: center;
+    box-sizing: border-box;
+    min-height: $iui-baseline * 3;
+    margin-left: calc(28px * (var(--level, 0)));
+    overflow: hidden;
+    padding-left: $iui-xxs;
+
+    &-icon {
+      @include iui-icons-default;
+      padding: 0 6px;
+      flex-shrink: 0;
+
+      &:first-child {
+        margin-left: $iui-expander-button-width;
+      }
+    }
+
+    &-expander {
+      > .iui-button-icon {
+        @media (prefers-reduced-motion: no-preference) {
+          transition: transform $iui-speed-fast ease-out;
+        }
+      }
+    }
+
+    &-label {
+      min-width: 0;
+      padding-left: 6px;
+
+      &:first-child {
+        margin-left: $iui-expander-button-width;
+      }
+    }
+
+    &-title,
+    &-caption {
+      white-space: nowrap;
+      overflow: hidden;
+      text-overflow: ellipsis;
+    }
+
+    &-title {
+      font-size: $iui-font-size;
+    }
+
+    &-caption {
+      font-size: $iui-font-size-small;
+      @include themed {
+        color: t(iui-text-color-muted);
+      }
+    }
+  }
+
+  &:focus {
```

Comment:
`:focus` will not work. When you add interaction, the focus will need to be handled manually (probably using `aria-activedescandant`), so you need a custom class name to emulate focus styling.

You can find all the info you need on W3C. [https://w3c.github.io/aria-practices/#:~:text=the%20tree%20role%20supports%20the%20aria-activedescendant%20property](https://w3c.github.io/aria-practices/#TreeView)

Reply:
Why we will need that attribute to handle focus?
![image](https://user-images.githubusercontent.com/36186912/144807159-2a325953-402f-4fec-9fad-4825134d5e1d.png)

Reply:
i thought we would need it because we have extra focusable elements checkboxes and buttons for every node, but now I'm thinking we could just set disable tabindex in those too

Reply:
The focus management will get a little complex here when we add this in React. W3C only has recommendations for expansion (using left/right arrow keys) and selection (using space/enter). Since we have checkboxes too, we will need to innovate here.

We can leave `:focus` here for now, but we should also add `:focus:not(:focus-visible)` at least. If we later find the need for a class like `.iui-focus`, we can come back and change it.

Reply:
Added `:focus:not(:focus-visible)`